reference/employer/openapi.yaml

```diff
@@ -0,0 +1,1083 @@
+                              nullable: true
+      summary: Employment
+      operationId: get-employment
+      tags:
+        - Employer
+      requestBody:
+        $ref: '#/components/requestBodies/Individual-Ids'
+  /employer/payment:
+    get:
+      description: Read payroll and contractor related payments by the company.
+      parameters:
+        - name: start_date
+          required: true
+          in: query
+          description: The start date to retrieve payments by a company (inclusive) in `YYYY-MM-DD` format.
+          schema:
+            type: string
+            format: date
+            example: '2021-01-01'
+        - name: end_date
+          required: true
+          in: query
+          description: The end date to retrieve payments by a company (inclusive) in `YYYY-MM-DD` format.
+          schema:
+            type: string
+            format: date
+            example: '2021-01-01'
+        - $ref: '#/components/parameters/API-Version'
+      responses:
+        '200':
+          description: Payment data
+          content:
+            application/json:
+              examples:
+                Success:
+                  value:
+                    - id: 20aa7cf2-949d-4d4e-9c01-499b59501ded
+                      pay_period:
+                        start_date: '2019-08-01'
+                        end_date: '2019-08-15'
+                      pay_date: '2019-08-22'
+                      debit_date: '2019-08-22'
+                      company_debit:
+                        amount: 5300000
+                        currency: usd
+                      gross_pay:
+                        amount: 4000000
+                        currency: usd
+                      net_pay:
+                        amount: 3200000
+                        currency: usd
+                      employer_taxes:
+                        amount: 200000
+                        currency: usd
+                      employee_taxes:
+                        amount: 350000
+                        currency: usd
+                      individual_ids:
+                        - 54719e14-5ea7-4fda-9898-f4d9ccb83c1a
+                        - 0ab620fc-203e-44ad-916c-facc69250f6f
+              schema:
+                type: array
+                items:
+                  type: object
+                  properties:
+                    id:
+                      type: string
+                      description: The unique id for the payment.
+                      pattern: '[0-9a-f]{8}[-]?(?:[0-9a-f]{4}[-]?){3}[0-9a-f]{12}'
+                    pay_period:
+                      type: object
+                      description: The pay period object.
+                      nullable: true
+                      properties:
+                        start_date:
+                          $ref: '#/components/schemas/Date'
+                        end_date:
+                          $ref: '#/components/schemas/Date'
+                    pay_date:
```

Comment:
pay_date, debit_date, company_debit, gross_pay, net_pay, employer_taxes, and employee_taxes are all nullable as well

Reply:
these are all refs, and the refs are nullable